### PR TITLE
Lab 2 CoinChanger

### PR DESCRIPTION
**Pull request by Sqaiser**
Professor please grade my lab 2 CoinChanger.

### CODE REVIEW

**Comment by khatchad** [comment]
There’s no grade in Blackboard?

**Comment by Sqaiser** [comment]
Hello Professor,

No not for this assignment.?

File: src/Lab2/CoinChanger.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ */
+import java.util.Scanner;
+public class CoinChanger {
+
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) {
+        // TODO: Ask the user for D
+  int quarters,Dimes, Nickles, Pennies,Cents,Totalcoins;
```
Comment (khatchad) [inline src/Lab2/CoinChanger.java:21]: Proper formatting needed. Also, variables begin with a lowercase letter.

File: src/Lab2/CoinChanger.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  System.out.print("Enter amount in change:");
+  Scanner Keyboard = new Scanner(System.in);
+  Cents = Keyboard.nextInt();
+  quarters =Cents /25;
+  Dimes=( Cents % 25 )/10;
+  Nickles = ( (Cents % 25)% 10)/5;
+  Pennies = (((Cents % 25)% 10)/5)/1;
```
Comment (khatchad) [inline src/Lab2/CoinChanger.java:28]: Interesting.

**Comment by khatchad** [comment]
:+1:

**Pull request closed without merging**